packages/widget_toolkit_pin/example/lib/main.dart

```diff
@@ -167,7 +170,9 @@ class AppPinCodeService implements PinCodeService {
 
   @override
   Future<String> encryptPinCode(String pinCode) async {
-    _pinCode = pinCode;
+    if (_pinCode == pinCode) {
```

Comment:
This check seems a little redundant, since essentially no matter the outcome we're always returning `Future.value(pinCode)`

Reply:
Removed